web/wp-content/themes/dctx-v2/template-parts/sharing-icons.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-i11ejt.php, but found phpcs-scan-I11ejT.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).